hoodie-client/src/main/java/com/uber/hoodie/HoodieWriteClient.java

```diff
@@ -353,10 +354,11 @@ public HoodieWriteClient(JavaSparkContext jsc, HoodieWriteConfig clientConfig,
     return updateIndexAndCommitIfNeeded(writeStatusRDD, table, commitTime);
   }
 
-  private void commitOnAutoCommit(String commitTime, JavaRDD<WriteStatus> resultRDD) {
+  private void commitOnAutoCommit(String commitTime, JavaRDD<WriteStatus> resultRDD,
```

Comment:
Changed the signature here instead of creating a new method just for compaction..
@vinothchandar ^